libkbfs/folder_branch_ops.go

```diff
@@ -1628,8 +1628,14 @@ func (fbo *folderBranchOps) SetInitialHeadFromServer(
 	}()
 
 	if md.IsReadable() && fbo.config.Mode().PrefetchWorkers() > 0 {
-		// We `Get` the root block to ensure downstream prefetches occur.
-		_ = fbo.config.BlockOps().BlockRetriever().Request(ctx,
+		// We `Get` the root block to ensure downstream prefetches
+		// occur.  Use a fresh context, in case `ctx` is canceled by
+		// the caller before we complete.
+		prefetchCtx := fbo.ctxWithFBOID(context.Background())
```

Comment:
Should we still have a timeout here?

Reply:
> Are you asking if we should add a timeout to the new context?

Yes that's what I meant. Sorry I didn't make it clear! I was a bit worried if not having a timeout here would cause goroutines hanging around when there are spotty network connections. I didn't look into the downstream code that uses this context though, so maybe we don't necessarily need it.

Reply:
Ah ok.  I think this will just end up using the normal block fetcher goroutines and not launch any new ones.

Reply:
Great. Thanks!